src/parser.js

```diff
@@ -70,13 +70,8 @@ define([
             title = head.children('h\\:title, title'),
             binds = head.find('bind'),
             instances = _getInstances(xml),
-            data = $(instances[0]).children();
-
-        xml.find('setvalue').each(function () {
-            var $this = $(this);
-            form.addSetValue(
-                $this.attr('event'), $this.attr('ref'), $this.attr('value'));
-        });
+            data = $(instances[0]).children(),
+            setValues = xml.find('setvalue');
```

Comment:
-1 for warning. Let's do correct form parsing instead. I did have a PR that did something like that, it's implemented on copy-paste-dev, but it's a dirty hack workaround for poor use of namespaces.